public/app/screens/main/sections/device-list/device-finder/device-finder.jsx

```diff
@@ -65,26 +91,35 @@ var DeviceFinder = React.createClass({
       action={this.startSearching} />;
   },
 
+  getDeviceFoundDialog: function getNoResultsDialog () {
```

Comment:
I would propose either use `=>` or remove the validation for method names and use:
```
getDeviceFoundDialog () {

}
```